src/app/pages/table-page/expandable-table.component.ts

```diff
@@ -9,7 +9,7 @@ import {KeyValuePipe, NgForOf, NgIf} from '@angular/common'
   selector: 'app-expandable-table',
   template: `
     <h1>Expandable Table</h1>
-    <tds-table enable-expandable-rows vertical-dividers="false" compact-design="false" enable-responsive="false">
+    <tds-table expandable-rows vertical-dividers="false" compact-design="false" responsive="false">
       <tds-table-header>
```

Comment:
The mileage Table column title seems to be right-aligned. Also you can't hover the columns in the expandable rows Table, only the title (rather than the whole column) gets the hover effect
![image](https://github.com/scania-digital-design-system/tegel-angular-demo/assets/48509966/b053f1c4-0393-43f1-aa3a-8532b6010968)